drivers/slipdev/slipdev.c

```diff
@@ -80,6 +84,45 @@ static void _slip_rx_cb(void *arg, uint8_t byte)
         dev->state = SLIPDEV_STATE_STDIN;
         isrpipe_write_one(&stdin_isrpipe, byte);
         return;
+    case SLIPDEV_STATE_CONFIG:
+        switch (byte) {
+        case SLIPDEV_ESC:
+            dev->state = SLIPDEV_STATE_CONFIG_ESC;
+            break;
+        case SLIPDEV_END:
+            crb_end_chunk(&dev->rb_config, true);
+            dev->state = SLIPDEV_STATE_NONE;
+            thread_flags_set(thread_get(dev->coap_server_pid), 1);
+            break;
+        default:
+            /* discard frame if byte can't be added */
+            if (!crb_add_byte(&dev->rb_config, byte)) {
+                DEBUG("slipdev: rx buffer full, drop frame\n");
```

Comment:
I see that this sneaked in below as well but it is not a good idea to try to print something in

1. ISR context
2. A UART handler callback